sources/web/datalab/static/appbar.html

```diff
@@ -30,37 +25,40 @@
+    <div class="btn-group">
       <button class="toolbar-btn" data-toggle="dropdown" title="Help Links">
-        <i class="material-icons">open_in_new</i>
+        <i class="material-icons">help_outline</i>
       </button>
       <ul class="dropdown-menu dropdown-menu-right" style="width: 200px">
+        <li><a id="showHelpLink" style="display:none" href="#">Show Help</a></li>
         <li><a id="keyboardHelpLink" style="display:none" href="#">Keyboard Shortcuts</a></li>
         <li><a id="markdownHelpLink" style="display:none" href="#">Markdown Syntax</a></li>
         <li id="notebookHelpDivider" class="divider" style="display:none"></li>
-        <li><a href="https://cloud.google.com/bigquery/what-is-bigquery" target="_blank"><span>BigQuery&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></a></li>
-        <li><a href="https://cloud.google.com/storage/docs/overview" target="_blank"><span>Cloud Storage&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></a></li>
+        <li><a href="/tree/datalab/docs/Readme.ipynb" target="_blank"><span>Samples and Tutorials</span></a></li>
+        <li class="divider"></li>
+        <li><a href="https://cloud.google.com/bigquery/what-is-bigquery" target="_blank"><span>BigQuery</span></a></li>
+        <li><a href="https://cloud.google.com/storage/docs/overview" target="_blank"><span>Cloud Storage</span></a></li>
         <li class="divider"></li>
+        <li><a href="http://googledatalab.github.io/pydatalab/" target="_blank"><span>Datalab API</span></a></li>
         <li><a href="https://developers.google.com/chart/" target="_blank"><span>Google Charts</span></a></li>
         <li><a href="http://matplotlib.org/contents.html" target="_blank"><span>Matplotlib</span></a></li>
         <li><a href="http://docs.scipy.org/doc/numpy/reference/" target="_blank"><span>Numpy</span></a></li>
         <li><a href="http://pandas.pydata.org/pandas-docs/stable/" target="_blank"><span>Pandas</span></a></li>
         <li><a href="http://scikit-learn.org/stable/" target="_blank"><span>SciKit-Learn</span></a></li>
         <li><a href="http://tensorflow.org" target="_blank"><span>TensorFlow</span></a></li>
-        <li class="divider"></li>
-        <li><a href="/tree/datalab/docs" target="_blank"><span>Samples and Tutorials</span></a></li>
-        <li><a href="http://googledatalab.github.io/pydatalab/" target="_blank"><span>Datalab API</span></a></li>
       </ul>
     </div>
-    <div class="btn-group">
-      <button id="aboutButton" title="About Google Cloud Datalab" class="toolbar-btn">
-        <i class="material-icons">info_outline</i>
-      </button>
-    </div>
     <div id="accountDropdown" class="btn-group">
       <button id="accountDropdownButton" title="Account" class="toolbar-btn">
         <i class="material-icons">account_circle</i>
       </button>
-      <div class="dropdown-menu dropdown-menu-right" style="min-width: 250px;">
+      <div class="dropdown-menu dropdown-menu-right" style="min-width: 150px;">
```

Comment:
please keep this at 250px. I know it doesn't look ideal with left-aligned menu items, but 150 is too narrow.